modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -1706,7 +1706,7 @@ public CodegenProperty fromProperty(String name, Property p) {
             property.isFile = true;
         }
         if (p instanceof UUIDProperty) {
-            property.isString = true;
+            property.isUuid = true;
```

Comment:
this seems dangerous to me. You should at least keep the isString = true and add your line just after. But, that would be strange because it would be the first case where the "is" field could be true for 2 variables. For example, right now you can't be isString and isDate at the same time.

Reply:
I would also suggest keeping `isString = true` to make the change backward compatible as we don't have the resource to review all the templates due to this change.